doc/api/buffer.md

```diff
@@ -788,6 +788,29 @@ const buf = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]);
 
 A `TypeError` will be thrown if `array` is not an `Array`.
 
+`Buffer.from(array)` will also use the internal `Buffer` pool if `array.length`
+is less than or equal to half [`Buffer.poolSize`] just like
+[`Buffer.allocUnsafe()`] does.
```

Comment:
> will also use the internal `Buffer` pool if `array.length` is less than or equal to half [`Buffer.poolSize`]

This does not look correct, see:
```js
> Buffer.poolSize
8192
> Buffer.from('1'.repeat(Buffer.poolSize / 2)).buffer // not pooled
ArrayBuffer { byteLength: 4096 }
> Buffer.from('1'.repeat(Buffer.poolSize / 2 - 1)).buffer // pooled
ArrayBuffer { byteLength: 8192 }
```

I suggest to remove «if `array.length` is less than or equal to half [`Buffer.poolSize`]» part.